.travis.yml

```diff
@@ -27,8 +27,8 @@ cache:
 env:
   matrix:
     # Test builds first, since they don't upload things.
-    - TEST_GOAL=coverage
-    #- TEST_GOAL=performance
+    - TEST_GOAL=coverage TOXCORE=toktok/c-toxcore
+    - TEST_GOAL=performance TOXCORE=irungentoo/toxcore
```

Comment:
Because I really just want to see if it compiles, since irungentoo/toxcore is very stable (i.e. non-changing). I could add another separate goal, but it seems like a waste of resources.